docs/source/extension/extension_dev.rst

```diff
@@ -196,7 +195,7 @@ The extension package containing the theme plugin must include all static assets
 
 See the `JupyterLab Light Theme <https://github.com/jupyterlab/jupyterlab/tree/master/packages/theme-light-extension>`__ for an example.
 
-See the `TypeScript theme cookiecutter <https://github.com/jupyterlab/theme-cookiecutter>`__ for a quick start to developing a theme plugin.
+See the `TypeScript extension cookiecutter <https://github.com/jupyterlab/extension-cookiecutter-ts>`__ (choosing ``theme`` as ``kind`` ) for a quick start to developing a theme plugin.
```

Comment:
Thanks! Probably also worth updating line 88 in `docs/source/extension/extension_migration.rst` which still refers to theme-cookiecutter:

> The create-theme script has been removed. If you want to create a new theme extension, you should use the [Theme Cookiecutter](https://github.com/jupyterlab/theme-cookiecutter) instead.

Reply:
Good catch - I'll do